tools/cli/internal/openapi/openapi.go

```diff
@@ -34,6 +34,7 @@ type Spec struct {
 	Paths        *openapi3.Paths               `json:"paths" yaml:"paths"`
 	Components   *openapi3.Components          `json:"components,omitempty" yaml:"components,omitempty"`
 	ExternalDocs *openapi3.ExternalDocs        `json:"externalDocs,omitempty" yaml:"externalDocs,omitempty"`
+	Extensions   map[string]any                `json:"-" yaml:"-"`
```

Comment:
doesn't `-` means ignored? do you really mean to ignore this field?

Reply:
you are right, this does not work. I am adding the missing part here https://github.com/mongodb/openapi/pull/774